cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Good point, I'm going to call this main_page_spec, because the format of all the tickets seem to have tests for each different page (collections/granules/rules)